src/plots/ternary/layout/attributes.js

```diff
@@ -0,0 +1,24 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+
+module.exports = {
```

Comment:
I could go either way. I would actually kind of like this partitioning in cartesian (where we don't have it), since there are so many little files for supplyDefaults that make it hard to look through the directory, but in other simpler cases it does feel a bit too much.